Exercise_1.cpp

```diff
@@ -47,6 +62,13 @@ int main()
     s.push(20); 
     s.push(30); 
     cout << s.pop() << " Popped from stack\n"; 
+    cout << s.isEmpty() <<" Is is empty?\n";

```

Comment:
Overall Good Job! Please add comments and time and space complexities and review inline comments.